tests/framework/web/UrlRuleTest.php

```diff
@@ -51,8 +51,8 @@ public function testParseRequest()
                 $request->pathInfo = $test[0];
                 $expected = $test[1];
                 $result = $rule->parseRequest($manager, $request);
-                if ($expected === false) {
-                    $this->assertFalse($result, "Test#$i-$j: $name");
+                if (empty($expected)) {
```

Comment:
This check is for empty strings. Empty string and `false` means completely different thing in this case. See https://github.com/yiisoft/yii2/pull/12610/files#diff-96fb0ef29c09f20871090c41bf5b3839R527